appcues/src/main/java/com/appcues/ui/presentation/ViewPresenter.kt

```diff
@@ -40,6 +42,7 @@ internal abstract class ViewPresenter(
     private val appcuesViewTreeOwner: AppcuesViewTreeOwner by inject()
     private val appcuesConfig: AppcuesConfig by inject()
 
+    private var parentView: ViewGroup? = null
```

Comment:
Looking at this again after mulling it over I think its best to change this to a weakReference instead (avoid any future issue since its a view)

Reply:
yeah that sounds like a good safeguard